liota/lib/protocols/helix_protocol.py

```diff
@@ -147,5 +147,11 @@ def on_receive(self, msg):
         if msg["type"] == "connection_response":
             require_field(msg["body"], "result")
             return True
+
+        elif msg["type"] == "get_properties_response":
```

Comment:
old version needs it (while debugging, found it).
just tested with new version, which does not need.
Delete it.